test/integration/k8ssvc/ca.crt

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Even though these are only used for tests, I'd really rather not have key material stored in the repo. Can you re-jig the tests so that these certs are generated on-the-fly when necessary in `test/integration/datasources_k8s.bats`'s `startup` function?

Reply:
Yeah, it is awful. I created the generate.sh and the openssl.cnf to make the cert generation automatically, sadly it is hard to setup in the integration test. Currently there are 3 differents ways of running integration tests ( the docker for CircleCi, the docker for local testing, and directly without docker). It would be much easier if there werer only 1 way of running integration tests.

Reply:
The integration tests should run the same regardless of where they're running. If you call the cert generation script conditionally (i.e. skip if the certs exist already) from the `setup` function, it shouldn't be too big of an impact.